src/App.tsx

```diff
@@ -17,18 +71,40 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                selectValue={filterSelect}
+                onSelect={setFilterSelect}
+                searchValue={searchValue}
+                onSearch={setSearchValue}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoaded
```

Comment:
`isLoading` will be more meaningful, because it's process and we render the loader
```suggestion
              {isLoading
```